src/Calculator/Views/TitleBar.xaml.cpp

```diff
@@ -103,14 +99,69 @@ namespace CalculatorApp
+    {
+        Dispatcher->RunAsync(CoreDispatcherPriority::Normal, ref new DispatchedHandler([this]() {
+            SetTitleBarControlColors();
+        }));
     }
 
-    void TitleBar::SetTitleBar()
+    void TitleBar::SetTitleBarControlColors()
     {
-        Window::Current->SetTitleBar(this->LayoutRoot);
+        auto applicationView = ApplicationView::GetForCurrentView();
+        if (applicationView == nullptr) { return; }
+
+        auto applicationTitleBar = applicationView->TitleBar;
+        if (applicationTitleBar == nullptr) { return; }
+
+        if (m_accessibilitySettings->HighContrast)
+        {
+            //Reset to use default colors.
+            applicationTitleBar->ButtonBackgroundColor = nullptr;
+            applicationTitleBar->ButtonForegroundColor = nullptr;
+            applicationTitleBar->ButtonInactiveBackgroundColor = nullptr;
+            applicationTitleBar->ButtonInactiveForegroundColor = nullptr;
+            applicationTitleBar->ButtonHoverBackgroundColor = nullptr;
+            applicationTitleBar->ButtonHoverForegroundColor = nullptr;
+            applicationTitleBar->ButtonPressedBackgroundColor = nullptr;
+            applicationTitleBar->ButtonPressedForegroundColor = nullptr;
+        }
+        else
+        {
+            Color bgColor = Colors::Transparent;
+            Color fgColor = safe_cast<SolidColorBrush^>(Application::Current->Resources->Lookup("SystemControlPageTextBaseHighBrush"))->Color;
+            Color inactivefgColor = safe_cast<SolidColorBrush^>(Application::Current->Resources->Lookup("SystemControlForegroundChromeDisabledLowBrush"))->Color;
+            Color hoverbgColor = safe_cast<SolidColorBrush^>(Application::Current->Resources->Lookup("SystemControlBackgroundListLowBrush"))->Color;
+            Color hoverfgColor = safe_cast<SolidColorBrush^>(Application::Current->Resources->Lookup("SystemControlForegroundBaseHighBrush"))->Color;
+            Color pressedbgColor = safe_cast<SolidColorBrush^>(Application::Current->Resources->Lookup("SystemControlBackgroundListMediumBrush"))->Color;
+            Color pressedfgCoolor = safe_cast<SolidColorBrush^>(Application::Current->Resources->Lookup("SystemControlForegroundBaseHighBrush"))->Color;
+            applicationTitleBar->ButtonBackgroundColor = bgColor;
+            applicationTitleBar->ButtonForegroundColor = fgColor;
+            applicationTitleBar->ButtonInactiveBackgroundColor = bgColor;
+            applicationTitleBar->ButtonInactiveForegroundColor = inactivefgColor;
+            applicationTitleBar->ButtonHoverBackgroundColor = hoverbgColor;
+            applicationTitleBar->ButtonHoverForegroundColor = hoverfgColor;
+            applicationTitleBar->ButtonPressedBackgroundColor = pressedbgColor;
+            applicationTitleBar->ButtonPressedForegroundColor = pressedfgCoolor;
+        }
     }
+
 }
 
+void CalculatorApp::TitleBar::OnHighContrastChanged(_In_ AccessibilitySettings ^ /*sender*/, _In_ Object ^ /*args*/)
```

Comment:
Move these into the `CalculatorApp` namespace above.  Let's keep the style consistent to one linebreak between each function definition.